server/main.lua

```diff
@@ -851,7 +843,7 @@ RegisterNetEvent('qb-phone:server:TransferMoney', function(iban, amount)
             end
         else
             local moneyInfo = json.decode(result[1].money)
-            moneyInfo.bank = round((moneyInfo.bank + amount))
+            moneyInfo.bank = QBCore.QBShared.Round((moneyInfo.bank + amount))
```

Comment:
```suggestion
            moneyInfo.bank = QBCore.Shared.Round((moneyInfo.bank + amount))
```
`QBCore.QBShared` doesn't exist, use `QBCore.Shared` instead.